### PR TITLE
arrow_docs: Added Redirect entries for each article

### PR DESCRIPTION
**Pull request by danidee10**
This PR adds Redirect entries for each article to improve triggering.

It also removes the empty `redirects.txt` and `trigger_words.txt` files.

## People to notify
@moollaza 

## Testing & Review
To be completed by Language Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Code**
- [ ] Uniformly indented, well commented
- [ ] Fetch.sh and Parse.sh run without errors
- [ ] Output contains no blank lines, or multi-line entries
- [ ] Fathead Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/arrow_docs

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/fathead/arrow_docs/parse.py [hunk 1]
```diff
@@ -37,8 +37,11 @@ def yield_api_example(api_examples):
 
         abstract = clean_output(api_example.dd.p.text)
         abstract = abstract_fmt.format(abstract, code)
+        
+        # redirect_alias: => 'arrow.arrow.Arrow.clone' => arrow clone
+        redirect_alias = 'arrow ' + title.split('.')[-1]
 
-        yield [title, 'A', '', '', '', '', '', '', '', '', '', abstract, url]
```
Comment (moollaza) [inline lib/fathead/arrow_docs/parse.py]: @danidee10 you need to keep this line. You want to _add_ redirects to what we have. Not replace what we have with only redirects.

Comment (danidee10) [inline lib/fathead/arrow_docs/parse.py]: I'm a bit confused @moollaza should i add redirects for each article in `output.txt` below and leave the existing Articles as they are (Doesn't make much sense to me) 

Or should i use the `redirects.txt` file to add the redirects?

Comment (moollaza) [inline lib/fathead/arrow_docs/parse.py]: @danidee10 the article entry must be in the `output.txt` for the article to exist at all.

Redirect in the `output.txt` are only a way to create aliases for existing articles in the output.txt

Using redirects.txt may be a simpler approach but the idea is that redirects.txt are written manually.

If we can programatically generate redirects and add them to the output.txt that is ideal 👍 


-----

### Example

`output.txt`
```
arrow.arrow.Arrow.clone	A	....more content...
Arrow.clone	R	arrow.arrow.Arrow.clone
Arrow clone	R	arrow.arrow.Arrow.clone
....
```

With only the first line in the output the query "arrow.arrow.Arrow.clone" is the only thing that would return our result.
With the Redirect entires, the new aliases "Arrow.clone" and "Arrow clone" will also return our "arrow.arrow.Arrow.clone" result when queried

**Comment by danidee10** [comment]
@moollaza hope i got it right this time ? 😄

**Comment by pjhampton** [comment]
Hey @danidee10!

[DuckDuckHack is now in Maintenance Mode](https://duckduckhack.com/) and from now on, we are only accepting issues and PRs for essential bugs and bug fixes.

Unfortunately, this is non-essential and will be closed.

We appreciate you taking the time to contribute and apologize for not being able to include your work.

**Pull request closed without merging**